### PR TITLE
Ticket/591 Add cancer centers theme library and styles.

### PR DESCRIPTION
**Pull request by alifr**
Add cancer centers theme library. Remove whitespace from some outputs.

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig [hunk 1]
```diff
@@ -1,82 +1,71 @@
...
-              {% endif %}
               {% if content.field_institution_type %}
-              <br />
-              {{ content.field_institution_type }}
-              {% endif %}
+                <br/>{{ content.field_institution_type }}
+              {%- endif -%}
             </div>
-            {% if content.field_org_head_name %}
             <p>
```
Comment (sarinapadilla) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig]: I think there should still be an "if" around this p tag - the org_head_name and org_head_title fields aren't required, so they might both be empty. I believe it's best practice to only output the HTML tags if we need them, so we only want to output the p tags if org_head_name has a value. If that isn't the case, then the p tag isn't needed.

Comment (alifr) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig]: Done.

**Comment by bryanpizzillo** [comment]
@alifr Can you update your branch, the merge from blog posts caused a conflict.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig [hunk 2]
```diff
@@ -1,82 +1,73 @@
...
             {% endif %}
             {% if content.field_phone_label_1 and content.field_phone_number_1 %}
-            <p>
-              {{ content.field_phone_label_1 }}: <a href="tel:{{ content.field_phone_number_1 }}" rel="nofollow">{{ content.field_phone_number_1 }}</a>
-            </p>
+              <p>
+                {{ content.field_phone_label_1 }}: <a href="tel:{{ content.field_phone_number_1 }}" rel="nofollow">{{ content.field_phone_number_1 }}</a>
+              </p>
             {% endif %}
             {% if content.field_phone_label_2 and content.phone_number_2 %}
```
Comment (sarinapadilla) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig]: @alifr There is one more change that I just noticed needs to be made in addition to the merge. The second field here needs to be updated to content.field_phone_number_2.

Comment (alifr) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cancer_center/node--cgov-cancer-center--full.html.twig]: Good catch. I've updated the file.

**Comment by alifr** [comment]
@bryanpizzillo upstream/develop has been merged into this branch. Conflicts are resolved.

**Pull request merged**